### DISCUSSION TITLE
Where to put non-global components, aka partials?

### DISCUSSION THREAD

**Discussion by crotanite** [nuxt/nuxt#18277] [Questions]
If I want to seperate a page's content into seperate partials but NOT use them as global components, where is the best place to put these files?

**Comment by wobsoriano** [comment]
Personally, I like Nuxt's flat component directory. Whether it's a reusable or a one-time component. Just add a prefix. You can check this link https://vueschool.io/articles/vuejs-tutorials/how-to-structure-a-large-scale-vue-js-application/

It says there that:

- Base components should all start with the same prefix (like `Base` or `App`)
- Single instance components should begin with the prefix `The`
- Tightly coupled child components should be prefixed with their parent component's name
- Component names should begin with the most top level (usually general) words and end with the most specific
